### ISSUE TITLE
Add app with Gitter chat

### ISSUE THREAD

**Issue by hello-smile6** [os-js/OS.js#757]
I think this is what we need (copied from the project homepage).
```
    <script>
      ((window.gitter = {}).chat = {}).options = {
        room: 'os-js/OS.js'
      };
    </script>
    <script src="https://sidecar.gitter.im/dist/sidecar.v1.js" async defer></script>
```
@andersevenrud Please review to make sure I'm not wrong.
This can be based on the HTML Viewer engine and embedded in a page included with all bundles.

**Comment by andersevenrud** [comment]
There's no need to use this javascript snippet.

A new app can be generated via the CLI utility that contains an iframe, then the source would just have to be changed to `https://gitter.im/os-js/OS.js/~embed`.

As I mentioned on Gitter, I would like an app like this not to simply just contain an embedded iframer. It would be nice to also have some contact links and maybe even another embedded view with a user guide or something like that.

**Comment by andersevenrud** [comment]
> As I mentioned on Gitter, I would like an app like this not to simply just contain an embedded iframe to gitter. It would be nice to also have some contact links and maybe even another embedded view with a user guide or something like that.

Just to expand on this: it would make it more of a support type application. Since gitter is dependent on a github account it would be important to have alternate ways of getting a conversation going.

**Comment by hello-smile6** [comment]
Could you do a 5-tab interface with the user guide and each of the chat rooms? It would promote participation and get people to improve the project.

**Comment by hello-smile6** [comment]
> There's no need to use this javascript snippet.
> 
> A new app can be generated via the CLI utility that contains an iframe, then the source would just have to be changed to `https://gitter.im/os-js/OS.js/~embed`.
> 
> As I mentioned on Gitter, I would like an app like this not to simply just contain an embedded iframe to gitter. It would be nice to also have some contact links and maybe even another embedded view with a user guide or something like that.

Good idea. That was my inspiration. Could you upload that and integrate, @andersevenrud ?

**Comment by hello-smile6** [comment]
App name: "Help", maybe replace "Welcome" in demo and other versions, would be compatible in standalone, right?

**Comment by andersevenrud** [comment]
Hm. I'm not sure I see any advantages in doing that vs linking to the other rooms. If people want to stay for a chat I would think that doing that in Gitter (and not OS.js) would be a **way better** experience.

**Comment by andersevenrud** [comment]
>  I would think that doing that in Gitter (and not OS.js) would be a way better experience.

... because the embedded chat rooms is basically just a handicapped version of the Gitter site.

**Comment by hello-smile6** [comment]
> Hm. I'm not sure I see any advantages in doing that vs linking to the other rooms. If people want to stay for a chat I would think that doing that in Gitter (and not OS.js) would be a **way better** experience.

If this ever goes to a native OS, you would have to reboot your computer just to contribute. Even in other situations, it would be harder to access. The point is an independent OS, not a fake OS like those emulators. Therefore, I think embedded is the way to go.

**Comment by andersevenrud** [comment]
> App name: "Help", maybe replace "Welcome" in demo and other versions, would be compatible in standalone, right?

"Help", "Support" or anything among those lines. It would be compatible in standalone mode.

> you would have to reboot your computer just to contribute

Would it ? Because if it went that road you'd probably have all the tools required to contribute.

And this isn't about contribution; really, but more about having a quick to maybe get some help. As I said, the embedded version is a handicapped version of the "real deal". And if someone wants to stay for a conversation it would be so much better to just use it the "real way".

If your idea of this was to create a full blown chat application, then there's ways to integrate that into the chat application that I already provide. It's possible to bridge gitter conversations via some custom software to integrate it wherever you want. So basically a "native" way.

**Comment by hello-smile6** [comment]
> > App name: "Help", maybe replace "Welcome" in demo and other versions, would be compatible in standalone, right?
> 
> "Help", "Support" or anything among those lines. It would be compatible in standalone mode.
> 
> > you would have to reboot your computer just to contribute
> 
> Would it ? Because if it went that road you'd probably have all the tools required to contribute.
> 
> And this isn't about contribution; really, but more about having a quick to maybe get some help. As I said, the embedded version is a handicapped version of the "real deal". And if someone wants to stay for a conversation it would be so much better to just use it the "real way".
> 
> > Therefore, I think embedded is the way to go
> 
> If your idea of this was to create a full blown chat application, then there's ways to integrate that into the chat application that I already provide. It's possible to bridge gitter conversations via some custom software to integrate it wherever you want. So basically a "native" way.

What chat? Could you give me an example server and login through Gitter PM so I could see? There's no feature like that's in the demo or standalone.

**Comment by andersevenrud** [comment]
I don't have a Gitter bridge set up for purposes like this. The demo only provides the Strophe application -- no servers.
